From c3b28da7b1158a565e4eb742c0affb380aa98725 Mon Sep 17 00:00:00 2001
From: Alcatraz323 <alcatraz32323@gmail.com>
Date: Wed, 8 Dec 2021 10:15:31 +0800
Subject: [PATCH 09/24] aura: RazerParts: set DeviceSettingsActivity category
 to LAUNCHER

currently, it does not appear in Settings, so add to Launcher for access
---
 RazerParts/AndroidManifest.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/RazerParts/AndroidManifest.xml b/RazerParts/AndroidManifest.xml
index 4042744..c6dbe01 100755
--- a/RazerParts/AndroidManifest.xml
+++ b/RazerParts/AndroidManifest.xml
@@ -35,7 +35,7 @@
                 android:launchMode="singleTask">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
-                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.LAUNCHER" />
                 <action android:name="com.android.settings.action.EXTRA_SETTINGS" />
             </intent-filter>
             <meta-data android:name="com.android.settings.dismiss"
-- 
2.34.0

